area/area_blackbox_test.go

```diff
@@ -1,22 +1,20 @@
 package area_test
 
 import (
+	"context"
 	"strconv"
 	"testing"
 	"time"
 
-	"context"
-
 	"github.com/almighty/almighty-core/area"
+	localerror "github.com/almighty/almighty-core/errors"
```

Comment:
We import this as is. The standard errors package is somewhere imported as `stderrors`.